File: app/src/main/java/com/example/ign_app/model/data/Thumbnails.java
package com.example.ign_app.model.data;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Thumbnails {

    @SerializedName("url")
    @Expose
    private String url;

    @SerializedName("size")
    @Expose
    private String size;

    @SerializedName("width")
    @Expose
    private String width;

    @SerializedName("height")
    @Expose
    private String height;

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getSize() {
        return size;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public String getWidth() {
        return width;
    }

    public void setWidth(String width) {
        this.width = width;
    }

    public String getHeight() {
        return height;
    }

    public void setHeight(String height) {
        this.height = height;
    }

    @Override
    public String toString() {
        return "Thumbnails{" +
                "url='" + url + '\'' +
                ", size='" + size + '\'' +
                ", width='" + width + '\'' +
                ", height='" + height + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/ign_app/articlemodel/data/Thumbnails.kt
package com.example.ign_app.articlemodel.data;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

class Thumbnails {

    @SerializedName("url")
    @Expose
    var url:String? = null

    @SerializedName("size")
    @Expose
    var size:String? = null

    @SerializedName("width")
    @Expose
    var width:String? = null

    @SerializedName("height")
    @Expose
    var height:String?=null

    override fun toString():String {
        return "Thumbnails{" +
                "url='" + url + '\'' +
                ", size='" + size + '\'' +
                ", width='" + width + '\'' +
                ", height='" + height + '\'' +
                '}';
    }
}
